script.js

```diff
@@ -0,0 +1,155 @@
+const mobileMenu = document.getElementById('mobile-menu');
+const menu = document.getElementById('menu');
+const close = document.getElementById('close-btn');
+const mobileLinks = document.querySelectorAll('.nav-list');
+
+function openMenu() {
+  mobileMenu.style.display = 'flex';
+}
+
+function closeMenu() {
+  mobileMenu.style.display = 'none';
+}
+
+menu.addEventListener('click', openMenu);
+close.addEventListener('click', closeMenu);
+
+mobileLinks.forEach((link) => {
+  link.addEventListener('click', closeMenu);
+});
+
+const speakerData = [
+  {
+    img: './images/speaker_01.png',
+    name: 'Yochai Benkler',
+    profession: 'Berkman Professor, Harvard Law.',
+    description: 'Yochai Benkler is an Israeli-American author and the Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School. He is also a faculty co-director of the Berkman Klein Center for Internet & Society at Harvard University',
+  },
+  {
+    img: './images/speaker_02.png',
```

Comment:
- [ ] please in the speaker section of the mobile version, make the picture bigger and align them with the text like in the original design. 
yours
![Screenshot (1100)](https://github.com/erickma1/CapstoneProject/assets/65298048/d1f7b3fb-cedd-4652-bd46-38a607b02f66)
original
![Screenshot (1099)](https://github.com/erickma1/CapstoneProject/assets/65298048/700d8e7a-d52e-463d-9c68-86a1b25068d2)